e2e/cypress/integration/messaging/mention_autocomplete_overlap_spec.js

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18667-At-mention user autocomplete does not overlap with channel header when drafting a long message containing a file attachment (standard viewport)', () => {
+        // # Upload file
+        cy.fileUpload('#fileUploadInput');
+
+        // # Create and then type message to use
+        cy.get('#post_textbox').clear();
+        let message = 'h{shift}';
+        for (let i = 0; i < 12; i++) {
+            message += '{enter}h';
+        }
+        cy.get('#post_textbox').type(message);
+
+        // # Add the mention
+        cy.get('#post_textbox').type('{shift}{enter}').type('@');
+
+        cy.get('#suggestionList').then((list) => {
+            cy.get('#channel-header').then((header) => {
+                // # Wait for suggestions to be fully loaded
+                cy.wait(TIMEOUTS.TINY).then(() => {
+                    // * Check overlap
+                    expect(list[0].getBoundingClientRect().top).to.be.at.least(header[0].getBoundingClientRect().bottom);
```

Comment:
Sorry for the delay. On the default Cypress window, the expectation is for the suggestion list to render visibly in the view.  So, I suggest to change it as
```suggestion
                    expect(list[0].getBoundingClientRect().top).to.be.at.least(header[0].getBoundingClientRect().bottom);
```

Reply:
Thanks for the comment @saturninoabril , but I did not understand it, mainly because for some reason there is no suggested change (my line and yours are the same).

If the suggestion list is to render visible in the view, I understand that the top of the list must have a client height >= 0 (inside the visible area). Am I right?

Reply:
Lol, I just realized I pasted the wrong line. Yup, that's right as you suggested.

Reply:
@larkox Not sure if it's clear but please update per suggestion, so we can merge, thanks

Reply:
Per DM, I did an update since @larkox is currently on travel.